zookeeper-server/src/main/java/org/apache/zookeeper/server/DataTree.java

```diff
@@ -241,13 +240,14 @@ public int getEphemeralsCount() {
      */
     public long approximateDataSize() {
         long result = 0;
-        for (Map.Entry<String, DataNode> entry : nodes.entrySet()) {
-            DataNode value = entry.getValue();
-            synchronized (value) {
-                result += getNodeSize(entry.getKey(), value.data);
-            }
-        }
-        return result;
+		
+		for (Map.Entry<String, DataNode> entry : nodes.entrySet()) { 
+			DataNode value =entry.getValue(); 
+			synchronized (value) { 
+				result +=getNodeSize(entry.getKey(), value.data); 
+				} 
+			} 
+		return result;
     }
 
```

Comment:
why need this?

Reply:
I was trying to use a stream here but couldn't come up with the correct approach and reverted back to the original for loop. I'll remove the extra indendations.